<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOctreeDialog</class>
 <widget class="QWidget" name="QOctreeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>八叉树</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>分辨率：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="input_resolution">
       <property name="maximum">
        <double>99999.990000000005239</double>
       </property>
       <property name="value">
        <double>128.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="input_X">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="input_Y">
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Z：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="input_Z">
       <property name="maximum">
        <double>99999.990000000005239</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_color">
     <property name="text">
      <string>搜索点颜色</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_vSearch">
       <property name="text">
        <string>体素搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_kSearch">
       <property name="text">
        <string>k邻域搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_rSearch">
       <property name="text">
        <string>半径邻域搜索</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
